Automattic/jetpack
# Blog Stats Block: Add Transform from Legacy Widget

Follow-up to #34557 

## Proposed changes:

* Allows the existing Blog Stats widget to be converted into the new block, which also has superior customisation abilities 

## Jetpack product discussion


N/A.

## Does this pull request change what data or activity we track or use?


No.

## Testing instructions:






You'll want to remove this first so that the Legacy Widget still appears:

https://github.com/Automattic/jetpack/blob/e94d76d165ae335e6038cafbe33e79a2fed220c3/projects/plugins/jetpack/modules/widgets/blog-stats.php#L44-L56

You can then add it to your site, and confirm that you can transform the block (press the calendar icon in the top left). The label should stay the same, and it should create a heading if you already have a widget title.

| <img width="280" alt="Screenshot 2024-02-02 at 09 07 35" src="https://github.com/Automattic/jetpack/assets/43215253/02003f3f-6970-4a9f-b6cf-33e5eeec933f"> | <img width="332" alt="Screenshot 2024-02-02 at 09 07 39" src="https://github.com/Automattic/jetpack/assets/43215253/b42a3457-46c7-472d-b9f6-a70df75c2a16"> | <img width="288" alt="Screenshot 2024-02-02 at 09 07 46" src="https://github.com/Automattic/jetpack/assets/43215253/c35a9bec-ad15-4228-8a4a-1a366110787e"> | 
|--------|--------|--------|

cc @kangzj - sorry, should have added it to start with but #34717 reminded me!
